homeassistant/components/smappee/strings.json

```diff
@@ -1,13 +1,33 @@
 {
     "config": {
+        "flow_title": "Smappee: {name}",
         "step": {
+            "environment": {
+                "description": "Set up your Smappee to integrate with Home Assistant.",
+                "data": {
+                    "environment": "Environment"
+                }
+            },
+            "local": {
+                "description": "Enter the host to initiate the Smappee local integration",
+                "data": {
+                    "host": "Host"
+                }
+            },
+            "zeroconf_confirm": {
+                "description": "Do you want to add the Smappee device with serialnumber `{serialnumber}` to Home Assistant?",
+                "title": "Discovered Smappee device"
+            },
             "pick_implementation": {
                 "title": "Pick Authentication Method"
             }
         },
         "abort": {
+            "already_configured": "The Smappee Cloud integration is already configured.",
```

Comment:
Please reference the common string. This abort reason will be used for all unique id aborts not only for the cloud entry abort.

https://github.com/home-assistant/core/blob/8258dcf41deaa334448fca19d20eaaf609c8d05a/homeassistant/strings.json#L52

Here's an example how to reference strings:
https://github.com/home-assistant/core/blob/8258dcf41deaa334448fca19d20eaaf609c8d05a/homeassistant/components/adguard/strings.json#L7